test/jasmine/tests/calcdata_test.js

```diff
@@ -886,7 +886,7 @@ describe('calculated data and points', function() {
             });
 
             // excludedTraces are traces that do not support sorting by value
-            var excludedTraces = [ 'carpet', 'contourcarpet'];
+            var excludedTraces = [ 'carpet', 'contourcarpet', 'image'];
```

Comment:
Can you turn this logic into a new trace-module category, just like

https://github.com/plotly/plotly.js/blob/e86c95b4b2abe646d7ab4e311fcd40cc61f1eaea/src/traces/image/index.js#L23

Naming it `'noSortingByValue'` would be good enough.

Reply:
Very nice idea! Done in e7eeb38b53bb358222880329af43962fe92142e9

Reply:
Thanks!!